Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
         .library(name: "JobsRedis", targets: ["JobsRedis"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/hummingbird-project/swift-jobs.git", from: "1.0.0-beta.1"),
+        .package(url: "https://github.com/thoven87/swift-jobs.git", branch: "stevenson/queued-at"),
```

Comment:
TODO: Update after swift jobs [PR](https://github.com/hummingbird-project/swift-jobs/pull/11) is merged